CMakeLists.txt

```diff
@@ -306,6 +316,9 @@ add_subdirectory(third_party)
 # Configure lit tests
 add_subdirectory(test/lit)
 
+# Configure GTest unit tests
```

Comment:
Is there an option build without tests?  Should there be?

Reply:
WABT has an option for that, but we don't. I'm not sure it would be valuable, at least for now; building the tests is a drop in the bucket compared to building everything else. Maybe if we eventually have a very large number of tests it will make sense to add an option to skip building them.

Reply:
Maybe lets wait until somebody requests that option?

Reply:
Hah, I will be the first one to say that I use tar.gz archive from github to download the source code for packaging emscripten based on the commits. https://chromium.googlesource.com/emscripten-releases/+/main/DEPS

The standalone tar.gz has been working fine until now. Guess I need to switch to git to get the source code or patch the CMakeLists.txt. No biggie.

Reply:
Sorry about chiming in here, I think it is always valuable to have a option to not build tests, in my case, it would help me to reduce the gtest dependency on the homebrew side. 

- Relates to https://github.com/Homebrew/homebrew-core/pull/97281

Reply:
Sounds good. A few people have asked for this now, so here it is: https://github.com/WebAssembly/binaryen/pull/4536.

Reply:
Thanks @tlively!

Reply:
Also, you want to check about the `latest release` thing, it still points to `105`. Thanks!